app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect, useRef } from "react";
+import {
+  Checkbox,
+  IconButton,
+  InputBase,
+  ButtonBase,
+  Popper,
+  fade,
+  makeStyles,
+  createStyles,
+} from "@material-ui/core";
+import Autocomplete, {
+  AutocompleteCloseReason,
+} from "@material-ui/lab/Autocomplete";
+import { CloseIcon, ExpandMore } from "../../components/Icons";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    popper: {
+      borderRadius: theme.shape.borderRadius,
+      borderWidth: 1,
+      borderStyle: "solid",
+      borderColor: "rgba(0, 0, 0, 0.23)",
+      boxShadow: theme.shadows[3],
+      zIndex: 1,
+      backgroundColor: theme.palette.background.default,
+      marginTop: theme.spacing(1),
+    },
+    button: {
+      borderRadius: theme.shape.borderRadius,
+      borderColor: "rgba(0, 0, 0, 0.23)",
+      borderWidth: 1,
+      borderStyle: "solid",
+      padding: "18.5px 14px",
```

Comment:
```suggestion
      padding: theme.spacing(2),
```
Would this still look ok? I think we should ideally avoid loose px values for consistency unless there's a good reason to make an exception.

Reply:
This is only here to match the default outlined inputs in the rest of form, which seemed to have [hard-coded padding](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/OutlinedInput/OutlinedInput.js). Completely agree that we should use spacing instead. I think I'll do this in another PR? Or is it such a small change anyway?

Sorry it was not included in the uiux PR because at the time this was not defined. From the mobile hi-fi, it is indeed 16px.

Reply:
It could be a good idea for these MuI hard coded values to put "our" values in the theme file. I'm pretty sure there's a way for it to apply to the default components using `theme.overrides`. Maybe something for a different PR

Reply:
Will leave these values for now. When we have hi-fi for profile editing, I'll do the extraction/override.